scrapy/__init__.py

```diff
@@ -31,7 +31,7 @@
 
 def __getattr__(name: str):
     if name == "twisted_version":
-        import warnings
+        import warnings  # pylint: disable=reimported
```

Comment:
I assume it's some magic around `__getattr__` that doesn't execute the top level imports but I didn't check.